src/main/java/io/axoniq/axonserver/connector/impl/ControlChannelImpl.java

```diff
@@ -197,14 +198,14 @@ private void registerOutboundStream(CallStreamObserver<PlatformInboundInstructio
     }
 
     @Override
-    public void reconnect() {
+    public synchronized void reconnect() {
```

Comment:
Can you elaborate why the `reconnect()` and `disconnect` methods should now be `synchronized`?